spec/tasks/rake_spec.rb

```diff
@@ -11,15 +11,15 @@
 
     it "creates a file" do
       run_task "hyrax:user:list_emails"
-      expect(File.exist?("user_emails.txt")).to be_truthy
+      expect(File).to be_exist("user_emails.txt")
```

Comment:
"to be exist" is more awkward than what was there before. :(

Reply:
Rubocop did that, and I don't like it either. Gross!